configs/_base_/datasets/cifar10.py

```diff
@@ -0,0 +1,34 @@
+dataset_type = 'mmcls::CIFAR10'
+
+# different from mmcls, we adopt the setting used in BigGAN
+# Note that the pipelines below are from MMClassification. Importantly, the
+# `to_rgb` is set to `True` to convert image to BGR orders. The default order
+# in Cifar10 is RGB. Thus, we have to convert it to BGR.
+img_norm_cfg = dict(
+    mean=[127.5, 127.5, 127.5], std=[127.5, 127.5, 127.5], to_rgb=True)
+train_pipeline = [
+    dict(type='RandomCrop', size=32, padding=4),
```

Comment:
`RandomCrop` has been supported in `MMClassification`. Because we adopt the dataset in `mmcls`. The `compose` function in `mmcls` will apply the pipelines implemented in `mmcls`.